### ISSUE TITLE
Architect table and other machines missing recipe on NEI

### ISSUE THREAD

**Issue by leterio** [BuildCraft/BuildCraft#4319]
BuildCraft version: 7.99.20
Forge version: 14.23.5.2768
Singleplayer or multiplayer: Both
Steps to reproduce: Start a singleplayer game

**Comment by AlexIIL** [comment]
Yes, currently the builder system is unfinished and has a few dupe bugs so none of it is craftable. However it's still functional in creative mode.
